README.md

```diff
@@ -1 +1,104 @@
+1. Click **Add Platform**, and select **Web API**
+1. Click **Save**
+
+> Note: When you add a *Web API* the Application registration portal, it adds a pre-defined App Id URI and Scope, using the format *api://{Application Id}/{Scope Name}* named **access_as_user** (you can review it by clicking 'Edit' button). This sample code uses this default scope.
+
+### Step 3: Configure your *TodoListService* project to match the Web API you just registered
+
+1. Open the solution in Visual Studio and then open the **Web.config** file under the root of **TodoListService** project.
+1. Replace the value of `ida:ClientId` parameter with the **Application Id** from the application you just registered in the Application Registration Portal.
+
+#### Step 3.1: Add the new scope to the *TodoListClient*`s app.config
+
+1. Open the **app.config** file located in **TodoListClient** project's root folder and then paste **Application Id** from the application you just registered for your *TodoListService* under `TodoListServiceScope` parameter, replacing the string `{Enter the Application Id of your TodoListService from the app registration portal}`. 
+
+    > Note: Make sure it uses has the format `api://{TodoListService-Application-Id}/access_as_user` (where {TodoListService-Application-Id} is the Guid representing the Application Id for your TodoListService).
+
+### Step 4: Register the *TodoListClient* application in the *Application registration portal*
+
+In this step, you configure your *TodoListClient* project by registering a new application in the Application registration portal. In the cases where the client and server are considered *the same application* you may also just reuse the same application registered in the 'Step 2.'.
+
+1. Go back to [Application registration portal](https://apps.dev.microsoft.com/portal/register-app) to register a new application
+1. Give a name to your Application, such as `NativeClient-DotNet-TodoListClient`, make sure that the *Guided Setup* option is **Unchecked** then press **Create**.
+1. Click **Add Platform**, and select **Native**.
+1. Click **Save**
+
+### Step 5: Configure your *TodoListClient* project
+
+1. In the *Application registration portal*, copy the value of the **Application Id**
+1. Open the **app.config** file located in the **TodoListClient** project's root folder and then paste the value in the `ida:ClientId` parameter value
+
+### Step 6: Run your project
+
+1. Press `<F5>` to run your project. Your *TodoListClient* should open.
+1. Select **Sign in** in the top right and sign in with the same user you have used to register your aplication, or a user in the same directory.
+1. At this point, if you are signing in for the first time, you may be prompted to consent to *TodoListService* Web Api.
+1. The sign-in also request the access token to the *access_as_user* scope to access *TodoListService* Web Api and manipulate the *To-Do* list.
+
+### Step 7: Pre-authorize your client application
+
```

Comment:
Why not do this step immediately after Step 5?

Reply:
In addition, pre-auth looks like - kind-of - optional to me - if you are building a single-tenant app.

Reply:
LGTM